index.bs

```diff
@@ -1233,8 +1232,43 @@ partial interface MLGraphBuilder {
     </div>
 </div>
 
+### hardSwish ### {#api-mlgraphbuilder-hard-swish}
+Computes the hard-swish activation that is introduced by [[MobileNetV3]] on the input tensor element-wise.
```

Comment:
Probably remove the word "hard swish" from the description.

Reply:
How about we say "Computes the h-swish nonlinearity that is introduced by [[MobileNetV3]] ... "? "h-swish nonlinearity" is the term used in the reference paper: https://arxiv.org/pdf/1905.02244.pdf

Reply:
I don't know enough that the term h-swish is a generic mathematical term of just a random name chosen randomly. But my point is that if there's a credible literature that backs the term and that it is widely known e.g. relu, then it's fine, but if not, it's probably better off just leaving the description as generic. @anssiko may have some opinion. It's just a nit though, you shouldn't be blocked by this.

Reply:
> it's probably better off just leaving the description as generic

Agree. I addressed it in the new commit. Please take another look. Thanks!

Reply:
The change that landed LGTM.

The best practice is to use generic language where possible and if using specialized language back it up with good reference(s) that define the new terms or concepts. The spec may also define new initially only locally used terms and concepts, and if there's a possibility of reuse these are to be annotated using the [definitions syntax](https://tabatkins.github.io/bikeshed/#definitions). [Autolinking](https://tabatkins.github.io/bikeshed/#autolinking) then creates cross-references to any existing definitions for reuse and checks for name clashes.

This is a good test: a reader who is familiar with the domain is able to understand the text without expanding references too much.